app/views/government_rulesets/show.html.erb

```diff
@@ -2,43 +2,102 @@
+      <h1 class="govuk-panel__title">
+        GOV.UK API Check Complete
+      </h1>
+      <div class="govuk-panel__body">
+        Score: <strong><%= @score.round(0) %> / 100</strong>
+      </div>
+    </div>
+    <br/><br/>
+    <h2 class="govuk-heading-m">Summary</h2>
+    <dl class="govuk-summary-list">
+      <% if @openapi %>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          OpenAPI Validity:
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @openapi.capitalize %>
+        </dd>
+      </div>
+      <% end %>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Issue Count:
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @counter %>
+        </dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Criticality:
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @criticality %>
+        </dd>
+      </div>
+    </dl>
+
+    <br/><br/>
+
+    <% if @issues.count > 0 %>
+    <table class="govuk-table linting-results-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">Results</caption>
       <thead class="govuk-table__head">
       <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header">Code</th>
-        <th scope="col" class="govuk-table__header">Line</th>
-        <th scope="col" class="govuk-table__header">Character</th>
-        <th scope="col" class="govuk-table__header">Problem</th>
+        <th scope="col" class="govuk-table__header">Criticality</th>
+        <th scope="col" class="govuk-table__header">Issue</th>
+        <th scope="col" class="govuk-table__header">Line Numbers</th>
         <th scope="col" class="govuk-table__header">Remedy</th>
       </tr>
       </thead>
       <tbody class="govuk-table__body">
-      <% @spectral_results.each do |result| %>
+      <% @issues.each do |issue| %>
         <tr class="govuk-table__row">
-          <th scope="row" class="govuk-table__header"><%= result['code'] %></th>
-          <td class="govuk-table__cell"><%= result['range']['start']['line'] %></td>
-          <td class="govuk-table__cell"><%= result['range']['start']['character'] %></td>
-          <td class="govuk-table__cell"><%= result['message'] %></td>
+          <td class="govuk-table__cell criticality-<%= issue[:criticality] %>"><%= issue[:criticality] %></td>
+          <td class="govuk-table__cell"><%= issue[:message] %></td>
+          <td class="govuk-table__cell">0</td>
           <td class="govuk-table__cell">
-            <a href="https://cddo-api-spec-tech-docs.herokuapp.com/api-linting-remediation.html ">Details</a>
-          </td>
+          <a href="<%= ENV["WIKI_URL"] %>">Details</a></td>
         </tr>
       <% end %>
       </tbody>
     </table>
+    <% end %>
 
     <div class="govuk-grid-row">
```

Comment:
[suggestion]
This is one of the places I'd use the no-print class:
```
<div class="govuk-grid-row no-print">
```
That way the three links won't appear when the page is printed.

Reply:
Added this.